lua/telescope/make_entry.lua

```diff
@@ -169,4 +169,29 @@ function make_entry.gen_from_quickfix(opts)
   end
 end
 
+function make_entry.gen_from_treesitter(opts)
+  opts = opts or {}
+  return function(entry)
+    local ts_utils = require('nvim-treesitter.ts_utils')
+    local start_row, start_col, end_row, end_col = ts_utils.get_node_range(entry.node)
+    local node_text = ts_utils.get_node_text(entry.node)[1]
+    local bufnr = vim.api.nvim_get_current_buf()
+    return {
+      valid = true,
+
+      value = entry.node,
+      ordinal = entry.kind .. " " .. node_text,
```

Comment:
I wonder if we should also add the text of the wrong it's on to sort on, because in large files, you might have a lot of the same variable name (like in Lua C files, you'll have `lstate` like.... 1000 times haha)

Reply:
not following this, what us "text of the wrong"? typo?

Reply:
haha, wow. that's not a great sentence.

"Text of the line" is what I meant to say -- not just the node text. Because sometimes it's hard to differentiate between all the different things with the same name.